src/cyw43_ll.c

```diff
@@ -345,40 +345,10 @@ static void cyw43_write_backplane(cyw43_int_t *self, uint32_t addr, size_t size,
     if (size == 4) {
         addr |= SBSDIO_SB_ACCESS_2_4B_FLAG;
     }
-    cyw43_write_reg(self, BACKPLANE_FUNCTION, addr, size, val);
+    cyw43_write_reg(self, BACKPLANE_FUNCTION, addr | SBSDIO_SB_ACCESS_2_4B_FLAG, size, val);
     cyw43_set_backplane_window(self, CHIPCOMMON_BASE_ADDRESS);
 }
 
-#if CYW43_ENABLE_BLUETOOTH
-static int cyw43_write_backplane_mem(cyw43_int_t *self, uint32_t addr, uint32_t len, const uint8_t *buf) {
```

Comment:
I'm happy with this change (to remove these functions and put them inline in `cyw43_ll_write_backplane_mem`), but I'm curious why it's needed?  Or, why they were needed as separate functions to begin with?

Reply:
I added cyw43_write_backplane_mem when originally adding SPI BT support. Infineon got improved the code to get rid of them. To simplify the code I guess. It seemed like a reasonable change to me

Reply:
OK, thanks!